application/controllers/QuestionAdministrationController.php

```diff
@@ -595,11 +582,11 @@ public function actionGetAdvancedOptions(
         $iQuestionId = null,
         $sQuestionType = null,
         $returnArray = false, //todo see were this ajaxrequest is done and take out the parameter there and here
-        $question_template = 'core'
+        $themeName = 'core'
```

Comment:
Shouldn't this be questionThemeName?

Reply:
https://github.com/LimeSurvey/LimeSurvey/pull/1933/files#r657195896 themeName (in my opinion) everywhere

Reply:
My vote is on questionThemeName